lib/counter_culture/extensions.rb

```diff
@@ -36,7 +36,15 @@ def counter_culture(relation, options = {})
               if: -> (model) { !model.paranoia_destroyed? }
           end
 
-          after_update :_update_counts_after_update
+          after_update :_update_counts_after_update, if: -> (model) do
+            if model.respond_to?(:paranoia_destroyed?)
+              !model.paranoia_destroyed?
+            elsif defined?(Discard::Model) && model.class.include?(Discard::Model)
+              !model.discarded?
+            else
+              true
+            end
+          end
```

Comment:
Can you extract this into a method please? Call it `destroyed_for_counter_culture?` perhaps (and use it with an `unless`)—that way it would be easy to override.